races/android.md

```diff
@@ -1,3 +1,87 @@
 ---
 wrapper: "../../../js/components/wiki/StubArticle"
```

Comment:
you'll want to change the wrapper from StubArticle to BasicArticle since it's not a stub anymore